### ISSUE TITLE
Search Engine

### ISSUE THREAD

**Issue by erikperin** [pagekit/pagekit#650]
Hello, I need that feature...

## Search Module

There's no search module on pagekit, I think that an important feature for this great CMS.

**Comment by neicv** [comment]
Hello All !

**Search Module**  

First alpha release _**Comming soon**_

Now, i cleen up code and prepare send prject to GitHub.

**Comment by neicv** [comment]
**First release**

https://github.com/neicv/pagekit-search

**Comment by tux-86** [comment]
Unable to activate "friendlyit/search". A fatal error occurred.

**Comment by neicv** [comment]
tux-86 plz say about you server (php, database, system... etc)

I not have any problem on Ubuntu Server Edition, Denwer (windows), OpenServer (windows)

**Comment by tux-86** [comment]
PHP 5.6.23 (fpm)
MariaDB 10.1.14
nginx 1.11.2
OS Ubuntu 12.04

**Comment by neicv** [comment]
thx!

if you enable debug mode on site - see additional message like "Array to string conversion" ?

**Comment by arceliver** [comment]
Yeah, I got the same error.

**Comment by MalteScharenberg** [comment]
This bug is not Pagekit related. Please discuss it with the extension developer (https://github.com/neicv/pagekit-search).

**Comment by neicv** [comment]
Yes, Please discuss it in the (https://github.com/neicv/pagekit-search).

**Comment by neicv** [comment]
Now Search extension available in the built-in official Marketplace Pagekit.

**Comment by erikperin** [comment]
Would be a Native Extension

**Comment by humantex** [comment]
+1, on a native function as erikperin suggests.

Future core code changes have a way of messing with things like plug-ins/extensions for basic and necessary site functions, and 3rd party maintainers are left scrambling to find and fix bugs. I've seen it happen with Drupal, Joomla, and others often enough and it's a real pain to deal with when universally expected functionality breaks a site.

I have 2 sites that I'm developing and am struggling with the search plugin working. Kudos to neicv on taking the initiative to fill the void, but IMHO this needs to be core code.
